xbmc/dialogs/GUIDialogKeyboardGeneric.cpp

```diff
@@ -129,7 +176,9 @@ void CGUIDialogKeyboardGeneric::OnInitWindow()
 bool CGUIDialogKeyboardGeneric::OnAction(const CAction &action)
 {
   bool handled = true;
-  if (action.GetID() == ACTION_ENTER || (m_isKeyboardNavigationMode && action.GetID() == ACTION_SELECT_ITEM))
+  if (action.GetID() == (KEY_VKEY | 8))
```

Comment:
Is there a define for what `8` means? Otherwise it's hard to understand what it means.